assets/platen.scss

```
@@ -0,0 +1,7 @@
+.book-columns.flatten-in-mobile {
+    flex-direction: column;
+
+    @media (min-width:56rem) {
+        flex-direction: row;
+    }
+}
```

Nit: missing trailing newline.
```suggestion
}

```